src/utils/translations/ta.ts

```diff
@@ -0,0 +1,102 @@
+const ta = {
+    // Home Tab
+    Home: "முகப்பு",
+    "Period in": "மாதவிடாய் காலம்",
+    "no info": "தகவல் இல்லை",
+    Period: "மாதவிடாய்",
+    today: "இன்று",
+    Delay: "தாமதம்",
+    Days: "(0)[நாள்];(1)[நாள்];(2-4)[நாள்];(5-20)[நாள்];(21)[நாள்];(22-24)[நாள்];(25-30)[நாள்];(31)[நாள்];(32-34)[நாள்];(35-40)[நாள்];",
```

Comment:
You can use instead:

```ts
    Days: "(0-inf)[நாள்]", 
```

Reply:
Okk , I Will Change That Properly

Reply:
what is That mean "inf",
  Days: "(0)[நாள்];(1)[நாள்];(2-4)[நாள்];(5-20)[நாள்];(21)[நாள்];(22-24)[நாள்];(25-30)[நாள்];(31)[நாள்];(32-34)[நாள்];(35-40)[நாள்];",  Is This Wrong Format ?

Reply:
This is right format, but you don't need to write all thisniptions are neccessary in case of language where specific word depends on current number

As example, in Russian we say `21 день` or `32 дня`.
In English there will be `1 day`, `2 days` etc.

In your language you don't have any difference and use the same word for every number, so it's unnecesaary to write the same for every option number and instead you can use (0-inf) what means from zero to infinity

Reply:
got it , I changed that , it works well, thank you guiding me :)